ts/frames.ts

```diff
@@ -1520,7 +1620,9 @@ export const cargoLimits = {
     },
     plantationcar_boxcar: {
         ['EFreightType::CrateTools']: 12,
-        ['EFreightType::CrudeOil']: 15,
```

Comment:
Are you sure plantation_boxcar can not hold crude oil? I think I added this because I saw a save file that was set to crude oil, but I'm not sure exactly which file it was.

Reply:
100% checked ingame